app/views/admin/configs/show.html.erb

```diff
@@ -134,56 +129,142 @@
-                                 placeholder: Constants::SiteConfig::DETAILS[:twitter_key][:placeholder] %>
-                <div class="alert alert-info"><%= Constants::SiteConfig::DETAILS[:twitter_key][:description] %></div>
-              </div>
-              <div class="form-group">
-                <%= admin_config_label :twitter_secret %>
-                <%= f.text_field :twitter_secret,
-                                 class: "form-control",
-                                 value: SiteConfig.twitter_secret,
-                                 placeholder: Constants::SiteConfig::DETAILS[:twitter_secret][:placeholder] %>
-                <div class="alert alert-info"><%= Constants::SiteConfig::DETAILS[:twitter_secret][:description] %></div>
-              </div>
-              <div class="form-group">
-                <%= admin_config_label :github_key %>
-                <%= f.text_field :github_key,
-                                 class: "form-control",
-                                 value: SiteConfig.github_key,
-                                 placeholder: Constants::SiteConfig::DETAILS[:github_key][:placeholder] %>
-                <div class="alert alert-info"><%= Constants::SiteConfig::DETAILS[:github_key][:description] %></div>
-              </div>
-              <div class="form-group">
-                <%= admin_config_label :github_secret %>
-                <%= f.text_field :github_secret,
-                                 class: "form-control",
-                                 value: SiteConfig.github_secret,
-                                 placeholder: Constants::SiteConfig::DETAILS[:github_secret][:placeholder] %>
-                <div class="alert alert-info"><%= Constants::SiteConfig::DETAILS[:github_secret][:description] %></div>
-              </div>
-              <div class="form-group">
-                <%= admin_config_label :facebook_key %>
-                <%= f.text_field :facebook_key,
-                                 class: "form-control",
-                                 value: SiteConfig.facebook_key,
-                                 placeholder: Constants::SiteConfig::DETAILS[:facebook_key][:placeholder] %>
-                <div class="alert alert-info"><%= Constants::SiteConfig::DETAILS[:facebook_key][:description] %></div>
-              </div>
-              <div class="form-group">
-                <%= admin_config_label :facebook_secret %>
-                <%= f.text_field :facebook_secret,
-                                 class: "form-control",
-                                 value: SiteConfig.facebook_secret,
-                                 placeholder: Constants::SiteConfig::DETAILS[:facebook_secret][:placeholder] %>
-                <div class="alert alert-info"><%= Constants::SiteConfig::DETAILS[:facebook_secret][:description] %></div>
+                Changing authentication keys will not take affect until devops restarts the app (about once per day). This will soon be instant.
```

Comment:
**Suggestion** Use more neutral language here, and don't promise future features!

```suggestion
                Changing authentication keys will not take affect until this forem instance is restarted.
```

Reply:
Done.